prow/plugins.yaml

```diff
@@ -22,6 +22,10 @@ slackevents:
   whitelist:
   - k8s-merge-robot
 
+label:
+  # github id for the milestone maintainers team
```

Comment:
How can I get this id for a team?

Reply:
it's kind of painful.  The only way I could manage it was by curling the teams endpoint as described here:

http://fabian-kostadinov.github.io/2015/01/16/how-to-find-a-github-team-id/

Reply:
Opened https://github.com/kubernetes/test-infra/pull/4543, ptal